croc.service

```diff
@@ -1,12 +1,55 @@
 [Unit]
-Description=croc relay
+Description=croc relay service
 After=network.target
 
 [Service]
+Environment="CROC_PASS=pass123"
```

Comment:
I believe I added this because the relay (and client) uses this password by default and a user may not notice/realise they can change it. By including it here, it's there to be noticed and changed.

Reply:
> the relay (and client) uses this password by default

I didn't know this was used as a default by croc itself. In that case, this makes sense. Thank you for clearing that up for me!